src/features/game/components/Syncing.tsx

```diff
@@ -14,8 +14,8 @@ export const Syncing: React.FC = () => {
       <span className="text-shadow text-xs text-center">
         Please bear with us while we sync all of your data on chain.
       </span>
-      <span className="text-shadow text-xs text-center underline mt-2">
-        Do not refresh your browser
```

Comment:
Would it be possible to remove the capital letters from this modal?

`Do not refresh your browser`